### PR TITLE
NMS-12520: Add documentation for downtime model delete attribute

### PR DESCRIPTION
**Pull request by indigo423**
### All Contributors

* [x] Have you read and followed our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [x] Have you [made an issue in the OpenNMS issue tracker](https://issues.opennms.org)?<br>If so, you should:
  1. update the title of this PR to be of the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
* [x] Have you made a comment in that issue which points back to this PR?
* [x] Have you updated the JIRA link at the bottom of this comment to link to your issue?
* [x] If this is a new or updated feature, is there documentation for the new behavior?
* [x] If this is new code, are there unit and/or integration tests?
* [x] If this PR targets a `foundation-*` branch, does it avoid changing files in `$OPENNMS_HOME/etc/`?

Add documentation for the downtime model change introduced in NMS-12442. We describe valid delete values for never, managed,always and when it is not set.

### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-12520

### CODE REVIEW

File: opennms-doc/guide-admin/src/asciidoc/text/service-assurance/downtime-model.adoc [hunk 1]
```diff
@@ -25,9 +25,22 @@ The scheduler changed the polling interval back to 5 minutes.
...
 <downtime interval="300000" begin="300000" end="43200000" /><!-- 5m, 5m, 12h --><2>
 <downtime interval="600000" begin="43200000" end="432000000" /><!-- 10m, 12h, 5d --><3>
-<downtime interval="3600000" begin="432000000" /><!-- 1h, 5d --><4>
+<downtime interval="3600000" begin="432000000" delete="never"/><!-- 1h, 5d --><4>
 ----
 <1> from 0 seconds after an outage is detected until 5 minutes the polling interval will be set to 30 seconds
 <2> after 5 minutes of an ongoing outage until 12 hours the polling interval will be set to 5 minutes
 <3> after 12 hours of an ongoing outage until 5 days the polling interval will be set to 10 minutes
-<4> after 5 days of an ongoing outage the service will polled only once a hour
+<4> after 5 days of an ongoing outage the service will polled only once a hour and we do not delete services
```
Comment (Bonrob2) [inline opennms-doc/guide-admin/src/asciidoc/text/service-assurance/downtime-model.adoc]:
```suggestion
<4> after 5 days of an ongoing outage the service will be polled only once a hour and we do not delete services
```

File: opennms-doc/guide-admin/src/asciidoc/text/service-assurance/downtime-model.adoc [hunk 2]
```diff
@@ -25,9 +25,22 @@ The scheduler changed the polling interval back to 5 minutes.
...
+
+The last downtime interval can have an attribute `delete` and allows you to influence the service lifecycle. 
+It defines the behavior that happens if a service doesn't come back online after 5 days.
+The following downtime attributes for `delete` can be used:
+
+[options="header, autowidth"]
+|===
+| Value     | Description
```
Comment (Bonrob2) [inline opennms-doc/guide-admin/src/asciidoc/text/service-assurance/downtime-model.adoc]:
```suggestion
| Value     | description
```

File: opennms-doc/guide-admin/src/asciidoc/text/service-assurance/downtime-model.adoc [hunk 3]
```diff
@@ -25,9 +25,22 @@ The scheduler changed the polling interval back to 5 minutes.
...
+| `never`   | services will never be deleted automatically
+| `managed` | only managed services will be deleted
+| `always`  | managed and unmanaged services will be deleted
+| _not set_ | If `delete` is not configured it is similar to `delete="never"` and is the default
```
Comment (Bonrob2) [inline opennms-doc/guide-admin/src/asciidoc/text/service-assurance/downtime-model.adoc]:
```suggestion
| _not set_ | if `delete` is not configured it is similar to `delete="never"` and is the default
```

File: opennms-doc/guide-admin/src/asciidoc/text/service-assurance/downtime-model.adoc [hunk 4]
```diff
@@ -25,9 +25,22 @@ The scheduler changed the polling interval back to 5 minutes.
 <downtime interval="30000" begin="0" end="300000" /><!-- 30s, 0, 5m --><1>
 <downtime interval="300000" begin="300000" end="43200000" /><!-- 5m, 5m, 12h --><2>
 <downtime interval="600000" begin="43200000" end="432000000" /><!-- 10m, 12h, 5d --><3>
-<downtime interval="3600000" begin="432000000" /><!-- 1h, 5d --><4>
+<downtime interval="3600000" begin="432000000" delete="never"/><!-- 1h, 5d --><4>
 ----
 <1> from 0 seconds after an outage is detected until 5 minutes the polling interval will be set to 30 seconds
```
Comment (Bonrob2) [inline opennms-doc/guide-admin/src/asciidoc/text/service-assurance/downtime-model.adoc]:
```suggestion
<1> from 0 seconds after an outage is detected until 5 minutes, the polling interval will be set to 30 seconds
```

File: opennms-doc/guide-admin/src/asciidoc/text/service-assurance/downtime-model.adoc [hunk 5]
```diff
@@ -25,9 +25,22 @@ The scheduler changed the polling interval back to 5 minutes.
...
 <2> after 5 minutes of an ongoing outage until 12 hours the polling interval will be set to 5 minutes
```
Comment (Bonrob2) [inline opennms-doc/guide-admin/src/asciidoc/text/service-assurance/downtime-model.adoc]:
```suggestion
<2> after 5 minutes of an ongoing outage until 12 hours, the polling interval will be set to 5 minutes
```

File: opennms-doc/guide-admin/src/asciidoc/text/service-assurance/downtime-model.adoc [hunk 6]
```diff
@@ -25,9 +25,22 @@ The scheduler changed the polling interval back to 5 minutes.
...
 <3> after 12 hours of an ongoing outage until 5 days the polling interval will be set to 10 minutes
```
Comment (Bonrob2) [inline opennms-doc/guide-admin/src/asciidoc/text/service-assurance/downtime-model.adoc]:
```suggestion
<3> after 12 hours of an ongoing outage until 5 days, the polling interval will be set to 10 minutes
```

**Comment by indigo423** [comment]
@Bonrob2 I've took over all your changes, let me know if there is more :) Thanks a lot for the review and the improvements.

**Comment by Bonrob2** [comment]
> @Bonrob2 I've took over all your changes, let me know if there is more :) Thanks a lot for the review and the improvements.

Nothing else, Ronny. You're most welcome!

**Pull request merged**